yt/frontends/ramses/hilbert.py

```diff
@@ -221,12 +192,8 @@ def get_cpu_list_cuboid(ds, X: "np.ndarray[Any, np.dtype[np.float64]]") -> set[i
         bounding_min[i] = omin * dkey
         bounding_max[i] = (omin + 1) * dkey
 
```

Comment:
The reason I am modifying those lines is because I line-profiled them, and 99.9% of the time was spent extracting those bound_keys! In fact, it is much faster to extract them from the dataset (as I now do above) once and for all.